tools/log_collection/collect_archive_logs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# save directory from which command is initiated
+collect_logs_directory=$(pwd)
```

Comment:
Ah, so should we be running this from tools?  If that's the case, and we're reworking this log collection script, we could just refactor it into 1 script and prompt for archive or current logs.

Reply:
> Ah, so should we be running this from tools?

@rh-dluong I think you would have to, and specifically from the same directory that the `excluded_files` portion of things is a part of.

For example, if you were do do this:

```console
# clarifying that we are in the root dir, basically where you would end up on a standard login
$ cd /root
$ /var/www/miq/vmdb/tools/log_collection/collect_archive_logs.sh
```

The above would be a valid way to execute the script from a `/bin/bash` perspective, but the way the script is written, the `tar` command would fail because it is expecting that `$(pwd)` is the directory that the `excluded_files` file is found, which would not be the case in the above invocation.

This probably wasn't a problem in the previous usage, because I assume the previous use case was to upload repo onto an appliance and then the script was executed after `cd`'ing into the directory it was uploaded to.  This workflow isn't made clear anywhere here or in that repo, so I am basing this on my interpretation of the code that is there.

Whether we solve this by refactoring or documentation is fine, but the lack of clarity is something that I think should be addressed before we move forward (at a minimum).

> If that's the case, and we're reworking this log collection script, we could just refactor it into 1 script and prompt for archive or current logs.

My PR:

https://github.com/redhat-cfme-support/cloudforms_4.X_log_collection/pull/15

Did that, and I just left the previous scripts there calling to it for legacy users who are used to that interface (the amount of code required for that is minimal, so it wasn't a big deal).